package.json

```diff
@@ -24,9 +23,10 @@
     "test:watch": "jest --watch",
     "prepublishOnly": "yarn lint && yarn test",
     "lint:eslint": "eslint . --cache --ext js,ts",
-    "lint:json": "prettier '**/*.json' --ignore-path .gitignore",
-    "lint": "yarn lint:eslint && yarn lint:json --check",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:json --write",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '**/*.yml' --single-quote --ignore-path .gitignore",
```

Comment:
This should be updated to omit `CHANGELOG.md`, which doesn't follow the same formatting rules

Reply:
Also, should we add a `.prettierrc` instead of specifying `--single-quote` here via the CLI?

Reply:
Done in: 47b3868